src/controllers/polygon/PolygonController.ts

```
@@ -134,9 +131,13 @@
    * @returns Schema Object
    */
   @Get(':did/:schemaId')
-  public async getSchemaById(@Path('did') did: string, @Path('schemaId') schemaId: string): Promise<unknown> {
+  public async getSchemaById(@Request() request: Req, @Path('did') did: string, @Path('schemaId') schemaId: string): Promise<unknown> {
```

ESLint

[error] Replace `@Request()·request:·Req,·@Path('did')·did:·string,·@Path('schemaId')·schemaId:·string` with `⏎····@Request()·request:·Req,⏎····@Path('did')·did:·string,⏎····@Path('schemaId')·schemaId:·string,⏎··`

(prettier/prettier)